### DISCUSSION TITLE
Duda valores P y K en el código base

### DISCUSSION THREAD

**Discussion by nonoam** [IIC2133-PUC/2024-2#28] [Tarea 0] [answered]
Hola, tengo un problema con los valores P y K, en cada iteración estas variables cambian sus valores (en ninguna parte modifico la memoria o valor de estos). Al principio pensé que era mi código que presentaba el error, pero el código base que nos presentan también genera este error. 

¿Alguien sabrá por qué ocurre esto?

Gracias de antemano

**Comment by waknaudt** [comment, marked as answer]
Hola, a mi me pasaba lo mismo, es un bug en el codigo base. Te recomiendo revisar https://github.com/IIC2133-PUC/2024-2/discussions/26
